provider/oci/common_test.go

```diff
@@ -118,19 +268,21 @@ type commonSuite struct {
 	fw      *ocitesting.MockOCIFirewallClient
 	storage *ocitesting.MockOCIStorageClient
 
-	env      *oci.Environ
-	provider environs.EnvironProvider
-	spec     environs.CloudSpec
-	// instance    instance.Instance
+	env         *oci.Environ
+	provider    environs.EnvironProvider
+	spec        environs.CloudSpec
+	config      *config.Config
 	ociInstance *ociCore.Instance
 	tags        map[string]string
+	ctrlTags    map[string]string
 }
 
 func (s *commonSuite) SetUpTest(c *gc.C) {
-	s.BaseSuite.SetUpTest(c)
+	s.IsolationSuite.SetUpTest(c)
 	oci.SetImageCache(&oci.ImageCache{})
+	*oci.WallClock = &advancingClock
```

Comment:
So I think it is cleaner to use object.SetClock() instead of setting a package global wallClock object.
I just would prefer that SetClock was defined in export_test.go rather than exposed always.

Reply:
Hmm, can you "export" a private method attached to a type? I could do that if it were possible.